pkgs/top-level/aliases.nix

```
@@ -344,6 +344,7 @@ mapAliases ({
   speedtest_cli = speedtest-cli;  # added 2015-02-17
   spice_gtk = spice-gtk; # added 2018-02-25
   spice_protocol = spice-protocol; # added 2018-02-25
+  spidermonkey_52 = throw "deprecrated 2019-10-16: use spidermonkey_60 instead";
```

I'm not sure I like these messages formatted like this.
It's confusing to people, was `spidermonkey_52` deprecated that date or did nixpkgs do this?

```suggestion
  spidermonkey_52 = throw "spidermonkey_52 has been removed. Please use spidermonkey_60 instead."; # added 2019-10-16
```